sklearn/cluster/k_means_.py

```diff
@@ -1262,6 +1264,15 @@ class MiniBatchKMeans(KMeans):
     Notes
     -----
     See http://www.eecs.tufts.edu/~dsculley/papers/fastkmeans.pdf
+
+    See also
```

Comment:
seems like it's generally inconsistent; I did a quick search and there were cases of both.

see also before notes: [neighbors/unsupervised.py](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/neighbors/unsupervised.py)

see also after notes: [linear_model/perceptron.py](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/linear_model/perceptron.py), [cross_decomposition/cca_.py](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/cross_decomposition/cca_.py)